### PR TITLE
Clarify Connection ID text

### PR DESCRIPTION
**Pull request by igorlord**
Clarify Connection ID use for 0-RTT and 1-RTT packets.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -670,7 +670,8 @@ field for a 0-RTT packet is selected by the client.
 The client can send 0-RTT packets after having received a packet from the server
 if that packet does not complete the handshake.  Even if the client receives a
 different connection ID from the server, it MUST NOT update the connection ID it
-uses for 0-RTT packets.  This enables consistent routing for all 0-RTT packets.
+uses for 0-RTT packets.  This ensures a predictable origin of connection IDs for
+all 0-RTT packets, see {{connection-id}}.
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md]: This phrasing doesn't make sense to me.

The point is that all 0-RTT are labelled with the same connection ID, therefore they (should) end up in the same place.

Comment (igorlord) [reply draft-ietf-quic-transport.md]: What I meant to say is that we are trying to make all packets of a particular type to always have CCID or SCID.  Mentioning "routing" here seems out of place -- network elements can use CID for whatever purpose they like, including routing.

I'll reword this.

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -697,17 +698,15 @@ as load balancers, to locate and use it.
...
 When the server receives a Client Initial packet and decides to proceed with the
 handshake, it chooses a new value for the connection ID and sends that in a
 Server Cleartext packet.  The server MAY choose to use the value that the client
 initially selects.
 
 Once the client receives the connection ID that the server has chosen, it uses
-this for all subsequent packets that it sends, except for any 0-RTT packets,
-which all have the same connection ID.
+this for all subsequent 1-RTT packets ({{packet-protected}}) and only 1-RTT
+packets that it sends.
```
Comment (igorlord) [inline draft-ietf-quic-transport.md]: Actually, this conflicts with # 558, which states that "Client Cleartext" also has SCID.  I'll fix that, too, and repost.

**Comment by igorlord** [comment]
Proposed connection ID clarifications, consistent with #588.

**Pull request merged**